src/coreclr/tools/aot/crossgen2/crossgen2.csproj

```diff
@@ -1,9 +1,48 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
   <PropertyGroup>
-    <OutputPath>$(RuntimeBinDir)/crossgen2</OutputPath>
-    <!-- The default value for macOS is false -->
-    <UseAppHost>true</UseAppHost>
-    <AppHostRuntimeIdentifier>$(Crossgen2PackageRID)</AppHostRuntimeIdentifier>
```

Comment:
If this is not used anymore (this is the usage), please delete the definition as well https://github.com/dotnet/runtime/blob/bb5d94d3d6d91f4b1bbadcb5ff923ac97684e61b/Directory.Build.props#L186-L188

Note that source build is not the only aspect which brings complication in RID calculation, there are several other corner-cases, platform and configurations (portable=true/false, cross=true/false, tizen, musl, etc.) which break something and usually show up _after_ the changes are merged. 😅